### ISSUE TITLE
Hiking Style Waterways

### ISSUE THREAD

**Issue by hungerburg** [andreynovikov/trekarta#79]
Waterways are very useful for orientation, in Trekarta they are rendered only very faint. Please make them wider (again).

![Screenshot_20190923-150848](https://user-images.githubusercontent.com/586972/65428596-f1dcc380-de14-11e9-8c6c-a0e88fdb828d.png)

If there was no ford, the "waterway | stream" would be very hard to spot!

**Comment by andreynovikov** [comment]
What do you mean by "again"? I didn't change their styling.

**Comment by hungerburg** [comment]
Indeed, looking at the screenshot in the other issue, its the same there. Source build now running, hell is my computer slow. A little digging in /app/src/main/assets/styles/maptrek.xml - applying width of 2 makes waterways just visible enough.
<style-line id="water" stroke="#a4bbcc" use="fix" width="2.0"/>

**Comment by andreynovikov** [comment]
Agree, would make it a little wider.

**Comment by hungerburg** [comment]
The line was eaten by the markup parser
`<style-line` id="water" stroke="#a4bbcc" use="fix" width="2.0"/>`

**Comment by hungerburg** [comment]
Hello Andrey, I do not see a difference in the latest release, do you?

![Screenshot_20191109-120618](https://user-images.githubusercontent.com/586972/68527671-d9076f00-02e9-11ea-9d57-41eff8873e48.png)

**Comment by andreynovikov** [comment]
As you can see from https://github.com/andreynovikov/trekarta/commit/15cd045bfe5e867365b1a87d4b5571aefeda9aac I've made it twice wider.

**Comment by hungerburg** [comment]
Indeed, I have to make it 4.0 to see waterways clearly.

**Comment by hungerburg** [comment]
Privjet Andrey, would you care? Subpixel changes seem no difference

**Comment by hungerburg** [comment]
Below screenshot, insignificant water made wider - I don't think it can be mistaken for an easy hiking path

![Screenshot_1574421354](https://user-images.githubusercontent.com/586972/69421975-c433ce00-0d22-11ea-832a-1ed1491e6d7e.png)

```patch
+++ app/src/main/assets/styles/inc_water.xml
                 <m cat="insignificant-water" v="stream">
-                    <line fade="13" use="water" width="1.0" />
+                    <line fade="13" use="water" width="4.0" />
                 </m>
                 <m cat="insignificant-water" v="ditch|drain">
-                    <line fade="14" use="water" width="0.3" />
+                    <line fade="14" use="water" width="2.0" />
                 </m>
```

**Comment by andreynovikov** [comment]
Fixed by 17d71e25417c215f289aaa45f562946cd3728a0a - streams are now drawn with scaling width line instead of fixed. And I absolutely do not want to enlarge ditches - they are very insignificant and there are lot of them in towns and agricultural fields.
